### PR TITLE
Add "Download PDF" button to North Star "Application Submitted" page

### PR DESCRIPTION
**Pull request by ssandbekkhaug**
### Description

Add "Download PDF" button to North Star "Application Submitted" page

## Release notes

n/a

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | under-development | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers)
- [x] Added an additional reviewer from outside your organization as FYI (if the primary reviewer is in the same organization as you)
- [x] Created unit and/or browser tests which fail without the change (if possible)
- [x] Performed manual testing (Chrome and Firefox if it includes front-end changes)
- [x] Extended the README / documentation, if necessary

#### User visible changes

- [x] Followed steps to [internationalize new strings](https://github.com/civiform/civiform/wiki/Internationalization-%28i18n%29#internationalization-for-application-strings)
  - [x] Added context strings to new [messages](https://github.com/civiform/civiform/blob/main/server/conf/i18n/messages)
  - [x] Didn't use a message in applicant facing code that isn't translated yet (unless behind a flag)
  - [x] Follow the workflow [here](<https://github.com/civiform/civiform/wiki/Internationalization-(i18n)#workflow-for-adding-or-editing-translations>)
- [x] Wrote browser tests using the [validateAccessibility](https://sourcegraph.com/github.com/civiform/civiform/-/blob/browser-test/src/support/index.ts?L437:14&subtree=true) method
- [x] Tested on mobile view. See [mobile device mode](https://developer.chrome.com/docs/devtools/device-mode/)
- [x] Manually tested at 200% size
- [x] Manually evaluated tab order

### Instructions for manual testing

1. Log in as client
2. Submit an application
3. View "Application Submitted" page
4. Click "Download PDF"

### Issue(s) this completes

#7762

### CODE REVIEW

File: server/app/views/applicant/ApplicantUpsellTemplate.html [hunk 1]
```diff
@@ -12,9 +12,15 @@ <h1 th:text="#{title.applicationConfirmation}"></h1>
...
         href="/"
-        class="usa-button"
+        class="usa-button margin-y-2"
         id="another-program"
         th:text="#{link.applyToAnotherProgram}"
+      ></a>
+      <a
+        th:href="${downloadHref}"
+        class="usa-button margin-y-2"
+        id="download-pdf"
```
Comment (greg-hyde) [inline server/app/views/applicant/ApplicantUpsellTemplate.html:22]: optional: you don't really need an ID on this button unless you're going to reference it in tests, so you could skip that part

**Review by greg-hyde — APPROVED** [review]

File: server/app/views/applicant/ApplicantUpsellTemplate.html [hunk 2]
```diff
@@ -12,9 +12,15 @@ <h1 th:text="#{title.applicationConfirmation}"></h1>
...
+        th:text="#{button.downloadPdf}"
```
Comment (swatkat1) [inline server/app/views/applicant/ApplicantUpsellTemplate.html:23]: Just a qq - are we planning on adding translations to the text? I understand this is still in works.

Comment (ssandbekkhaug) [reply server/app/views/applicant/ApplicantUpsellTemplate.html:22]: Yes, `#{button.downloadPdf}` means that the text will be translated

**Review by swatkat1 — APPROVED** [review]
Thank you for your work!

**Pull request merged**